qml/Installer.qml

```diff
@@ -71,7 +71,11 @@ Page {
             anchors.horizontalCenter: parent.horizontalCenter
             color: "gray"
             text: i18n.tr("Running")
-            onClicked: console.log("Installer is running")
+            onClicked: {
+                console.log("Installer is running")
+                if( startButtonFake.color == "#008000" )
```

Comment:
- don't use colors for conditions (visible is a better choice)
- try to match the code style of the rest of the project, eg.
```
if (somecondition) {
```
- always put braces around conditional expression's body
```
if (somecondition) {
  // do stuff
}
```

Reply:
it's always visible tho, only the color/text changes

Reply:
Using the activity component now xd